### ISSUE TITLE
Feature request #@var

### ISSUE THREAD

**Issue by Zaitsev** [Camelcade/Perl5-IDEA#1299]
Add annotation `#@var MyPackage::MooRoleA | MyPackage2:MooRoleB $variable` so it will be possible auto completion for _interfaces

**Comment by hurricup** [comment]
Do you mean this: https://github.com/Camelcade/Perl5-IDEA/issues/1248 ?
And have you read this: https://github.com/Camelcade/Perl5-IDEA/wiki/Perl-syntax-nuances ?

**Comment by Zaitsev** [comment]
Sorry - 312 isues is too hard to review them all......

**Comment by hurricup** [comment]
Yep :) Closing

**Comment by hurricup** [comment]
And atm you may use perl syntax `my Foo::Bar $var`

**Comment by Zaitsev** [comment]
Yes. Im using it. But very often it requires RoleA | ReleB....

**Comment by hurricup** [comment]
I'm not sure what is the case? If you have a class ClassC with RoleA and RoleB you may just use `my ClassC $var`.

**Comment by Zaitsev** [comment]
Yes. But what if var could be ClassA OR CalssB (returned by some function/factory)  and both classes implements RoleB AND RoleC. In such context var's Class  often irrelevant or even unknown but an interface is important.
